src/components/socials/Social.tsx

```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+import socialsimage from "@/public/social/socialsimage.webp";
+
+const SocialBanner = () => {
+  return (
+    <div className="relative flex w-full overflow-hidden">
```

Comment:
```suggestion
    <div className="w-full">
```
all those classes do nothing so remove them